src/fusor/examples/bcr_abl1.json

```diff
@@ -1,111 +1,131 @@
 {
   "type": "CategoricalFusion",
-  "structural_elements": [
-    {
-      "type": "TranscriptSegmentElement",
-      "transcript": "refseq:NM_004327.3",
-      "gene_descriptor": {
-        "type": "GeneDescriptor",
-        "id": "normalize.gene:BCR",
-        "gene_id": "hgnc:1014",
-        "label": "BCR"
+  "structure": {
+    "type": "Adjacency",
+    "adjoinedSequences": [{
+      "type": "SequenceLocation",
```

Comment:
The adjoinedSequence SequenceLocation objects for the adjacency are based on genomic coordinates corresponding to the fusion junction; for categorical fusions and many reported assayed fusions, these are typically at (or close to) transcript boundaries as aligned to a chromosome.